MAPL_ExceptionHandling/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+esma_set_this ()
+
+set (srcs
+  MAPL_ExceptionHandling.F90
+  MAPL_ErrorHandling.F90
+  MAPL_Throw.F90
+  )
+esma_add_library (${this} SRCS ${srcs} DEPENDENCIES MPI::MPI_Fortran)
+#esma_add_library (${this} SRCS ${srcs} DEPENDENCIES gftl-shared gftl MPI::MPI_Fortran)
+target_include_directories (${this} PRIVATE ${MAPL_SOURCE_DIR}/include)
+
+file (COPY MAPL_ErrLogMain.h DESTINATION ${esma_include}/${this})
+file (COPY MAPL_ErrLog.h DESTINATION ${esma_include}/${this})
+file (COPY MAPL_Exceptions.h DESTINATION ${esma_include}/${this})
```

Comment:
These should move into an include subdirectory under the main MAPL.

Then just add  cmake:   target_include_directories(${this}  ... )

Note that you'll need  a $<BUILD_INTERFACE...> and a $<INSTALL_INTERFACE ...> clause.

Reply:
You will also need to create a small CMakeLists.txt in the new include directory to install these files.

Reply:
@tclune I'm not sure we need `$<INSTALL_INTERFACE>` here, or are you forward-thinking? This [StackOverflow post](https://stackoverflow.com/a/58729533/1876449) seems to say that `$<INSTALL_INTERFACE>` is only used during `install(EXPORT)`.